src/aiokem/main.py

```diff
@@ -1,3 +1,92 @@
-def add(n1: int, n2: int) -> int:
-    """Add the arguments."""
-    return n1 + n2
+"""AioKem class for interacting with Kohler Energy Management System (KEM) API."""
+
+from typing import Any
+
+from aiohttp import ClientSession, ClientTimeout
+
+AUTHENTICATION_URL = "https://kohler-homeenergy.okta.com/oauth2/default/v1/token"
+CLIENT_KEY = (
+    "MG9hMXFpY3BkYWdLaXdFekYxZDg6d3Raa1FwNlY1T09vMW9"
+    "PcjhlSFJHTnFBWEY3azZJaXhtWGhINHZjcnU2TWwxSnRLUE5obXdsMEN1MGlnQkVIRg=="
+)
+API_KEY = "pgH7QzFHJx4w46fI~5Uzi4RvtTwlEXp"
+API_BASE = (
+    "https://az-amer-prod-hems-capp.jollyglacier-72224ec0.eastus.azurecontainerapps.io"
+)
```

Comment:
I wonder if we can fetch this URL from another more stable endpoint in case it changes?

Reply:
I don't see it anywhere in the traces or using aliases in DNSlookups, so I suspect it's hard-coded in the app.